File: src/ACT.Domain/Entities/Company.cs
namespace ACT.Domain.Entities;

public class Company
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string? ContactEmail { get; set; }
    public string? Phone { get; set; }
    public string? Address { get; set; }
    public ICollection<BrandSettings> BrandSettings { get; set; } = new List<BrandSettings>();
    public ICollection<Client> Clients { get; set; } = new List<Client>();
    public ICollection<TreatmentType> TreatmentTypes { get; set; } = new List<TreatmentType>();
    public ICollection<Treatment> Treatments { get; set; } = new List<Treatment>();
}

File: frontend/src/types/company.ts
export interface CompanyDto {
  id: number
  name: string
  contactEmail?: string
  phone?: string
  address?: string
}

export interface CreateCompanyRequest {
  name: string
  contactEmail?: string
  phone?: string
  address?: string
}

export interface UpdateCompanyRequest {
  name: string
  contactEmail?: string
  phone?: string
  address?: string
}

